experiments/enableTTL/README.md

```diff
@@ -0,0 +1,188 @@
+
+Navigate to CRX/de on your publish instance, and open `/apps/dispatchertester/config`: [localhost:4503/crx/de/index.jsp#/apps/dispatchertester/config](http://localhost:4503/crx/de/index.jsp#/apps/dispatchertester/config)
+
+Create a copy of the existing `DispatcherMaxAgeHeaderFilter` file and paste it next to the original. Rename it's suffix from `set-1` to `set-2`. Make sure to save your changes.
+
+Open the properties of your new node, and set the `filter.pattern` entry to:
+
+```
+/etc.clientlibs/dispatchertester/clientlibs/(.*)
+```
+
+Additionally, set `max.age` to `86400` (60 seconds in a minute * 60 minutes in an hour * 24 hours in a day = 86400 seconds in a day). This means that all cached items that exist under this path will be able to live in the cache for 24 hours.
+
+Make sure to save your changes in CRX/de. The new node should look like this:
+
+<img src="../img/crxde-new-config-node.png">
+
+> Note: typically this configuration would be part of the app's source code. We're adding it directly to CRX/de for the sake of speed only.
+
+### Test out the new TTL
+
+With the new Dispatcher TTL config in place on your publish instance, make a request to a known clientlib to see if the `max-age` is returned correctly:
+
+```
+curl -H "Server-Agent: Communique-Dispatcher" -D - http://localhost:4503/etc.clientlibs/dispatchertester/clientlibs/clientlib-site.js -o /dev/null
+```
+
+Among the headers listed, you should see `Cache-Control: max-age=86400` for this resource.
+
+To confirm we didn't globally change the TTL values for other files, try the original cURL request we made:
+
+```
+curl -H "Server-Agent: Communique-Dispatcher" -D - http://localhost:4503/content/dispatchertester/us/en.html -o /dev/null
+```
+
+Should return `Cache-Control: max-age=60`. 
+
+Excellent! We now have different TTL values set for different content paths.
+
+Opening the Dispatcher Test Harness "home" page in your browser should result in both the en.html page and it's clientlibs being written to the cache: [aem-publish.local:8080/content/dispatchertester/us/en.html](http://aem-publish.local:8080/content/dispatchertester/us/en.html). 
+
+Taking a look in the cache reveals the last modified timestamp of clientlib-site.js is correct, exactly 24 hours in the future:
+
+<img src="../img/24-hr-expiry.png">
+
+## Implications of enableTTL
+
+When the `enableTTL` flag is set and there is a corresponding `<filename>.ttl` file present in the cache, `.stat` files will not be considered when determining whether or not the requested `<filename>` is stale. 
+
+In other words, if the activation of page-A.html occurs which touches a `.stat` file at the root of the cache (assuming a `/statfileslevel` of 0), then page-B.html (a peer of page-A.html) **will not be re-fetched** until it's TTL expires. This is the case even though it's last modified timestamp is now older than it's nearest .stat file.
```

Comment:
This paragraph was a bit of a head scratcher but I'm not certain how to make it read more clear.  Also no apostrophe need on "it's" i.e. "This is the case even though its last modified timestamp is now older than its nearest..."